package.json

```
@@ -52,8 +52,8 @@
     "uuid": "9.0.0"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^7.0.0",
-    "@openedx/paragon": "^22.0.0",
+    "@edx/frontend-platform": "^7.0.0 || ^7.1.2-tokens.1",
+    "@openedx/paragon": ">= 22.0.0 < 24.0.0 || ^23.0.0-alpha.1",
```

[nit]: I think we can now remove 23.0.0-alpha.1 both here and in other PRs

```suggestion
    "@openedx/paragon": ">= 22.0.0 < 24.0.0",
```

---

Removed